homeassistant/components/switcher_kis/config_flow.py

```diff
@@ -39,10 +58,17 @@ async def async_step_confirm(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Handle user-confirmation of the config flow."""
+        if user_input is None:
+            return await self._show_setup_form(user_input)
```

Comment:
Isn't Param 2 for this function errors?
Also no need to pass it if you know it is None.

Reply:
?

Reply:
function removed for a simpler solution